libs/SalesforceSDK/src/com/salesforce/androidsdk/security/AppLockManager.kt

```diff
@@ -63,17 +63,17 @@ internal abstract class AppLockManager(
         EventsObservable.get().notifyEvent(EventsObservable.EventType.AppUnlocked)
     }
 
-    fun getAccountPrefs(account: UserAccount): SharedPreferences {
+    fun getAccountPrefs(account: UserAccount?): SharedPreferences {
         val ctx = SalesforceSDKManager.getInstance().appContext
-        return ctx.getSharedPreferences(policyKey + account.userLevelFilenameSuffix, Context.MODE_PRIVATE)
+        return ctx.getSharedPreferences(policyKey + account?.userLevelFilenameSuffix, Context.MODE_PRIVATE)
     }
 
     fun getGlobalPrefs(): SharedPreferences {
         val ctx = SalesforceSDKManager.getInstance().appContext
         return ctx.getSharedPreferences(policyKey, Context.MODE_PRIVATE)
     }
 
-    fun getPolicy(account: UserAccount): Policy {
+    fun getPolicy(account: UserAccount?): Policy {
```

Comment:
This is incorrect.  There are two places where this is API is used and both check the account is not null first.